### PR TITLE
add ability to recieve and send Audio

### PR DESCRIPTION
**Pull request by temagi**

### CODE REVIEW

File: examples/echo_bot.py [hunk 1]
```diff
@@ -49,7 +49,12 @@
 
 @dp.message_handler(regexp='^cat$')
 async def send_photo(message: Message):
-    await bot.send_photo(message.chat.id, 'images/cat.jpg')
+    await bot.send_media(message.chat.id, 'images/cat.jpg', "IMAGE")
```
Comment (yerassyl94) [inline examples/echo_bot.py]: please change string value "IMAGE" to typealias like here [https://github.com/btsdigital/pyAitu/blob/develop/pyAitu/models/form/input_type.py](url)

File: pyAitu/bot/bot.py [hunk 2]
```diff
@@ -77,15 +77,16 @@ class Bot(BaseBot):
         result = await self.request(EDIT_MESSAGE, payload)
         return result
 
-    async def send_photo(self,
+    async def send_media(self,
                          chat_id: str,
-                         photo: str):
-        result = await self.upload_file(photo)
+                         file: str,
+                         file_type: str):
```
Comment (yerassyl94) [inline pyAitu/bot/bot.py]: file_type wait here typealias instead of string

**Review by yerassyl94 — CHANGES_REQUESTED** [review]

File: examples/echo_bot.py (hunk 1 above)
Comment (temagi) [reply examples/echo_bot.py]: fixed

File: pyAitu/bot/bot.py (hunk 2 above)
Comment (temagi) [reply pyAitu/bot/bot.py]: fixed

**Review by yerassyl94 — APPROVED** [review]

**Pull request merged**